lib/LoginStrategy.types.ts

```diff
@@ -1,5 +1,6 @@
 import { IRoleDefinitionV2 } from '@energyweb/credential-governance';
 import { VerifiablePresentation } from '@ew-did-registry/credentials-interface';
+import type { SiweMessage as SiweMessagePayload } from 'siwe';
```

Comment:
what is the reason of creating alias?

Reply:
No specific reason, just to avoid confusion for the use case.